docs/core/docker/docker-basics-dotnet-core.md

```diff
@@ -33,11 +33,11 @@ You can build both Windows and Linux containers with [multi-arch based tags](htt
 
```

Comment:
Can you also remove lines 4 and 5 (author and ms.author metadata) and update ms.date to 10/11/2018?

Reply:
Done